src/lib/core/overlay/position/connected-position-strategy.ts

```diff
@@ -116,10 +117,11 @@ export class ConnectedPositionStrategy implements PositionStrategy {
       // (top, left) coordinate for the overlay at `pos`.
       let originPoint = this._getOriginConnectionPoint(originRect, pos);
       let overlayPoint = this._getOverlayPoint(originPoint, overlayRect, viewportRect, pos);
+      let overlayDimensions = this._getCSSDimensions(overlayRect, overlayPoint, pos);
```

Comment:
Done. FWIW, I had it in a separate method, because the logic used to be a bit longer.